.github/workflows-support/requirements.txt

```diff
@@ -0,0 +1,7 @@
+Cython==0.29.16
+Jinja2==2.11.1
+numpy==1.18.3
+Pillow==6.2.1
+Pygments==2.6.1
+pyreadline==2.1
```

Comment:
i'm not a fan of hardcoding versions, because we have to update those manually then. is there a better way to auto-update the libs and have the most recent one used?

Reply:
should work out without the versions as well as this gets just read by `pip -r` afaik